compose.yml

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: "3"
+
+services:
+  site:
+    build: "."
+    image: "ozone-site-dev"
+    ports:
+      - "3000:3000"
+    volumes:
+      - ".:/ozone-site"
+      - /ozone-site/node_modules/
```

Comment:
Let's add a comment explaining that this prevents the host's node_modules directory from being mounted if it exists, so that the one in the image is used.

Reply:
This does end up creating an empty node_modules directory in the host, but I think that is an ok trade off.

Reply:
Added the comment to clarify the usage of the line, thanks for the suggestion.
Now that I look at it, it would have caused confusion down the line to others.